content/articles/Async-js-using-ajax/index.md

```diff
@@ -0,0 +1,235 @@
+## Asynchronous JavaScript: Fetching Data from external API using AJAX
+
+### Introduction
+
+**AJAX** stands for Asynchronous JavaScript and XML, it’s a set of web technology used to send and receive data between a client and server. Ajax allows you to make requests asynchronously without having to reload the page. Ajax can send and receive information in various formats, including JSON, XML, HTML, and text files. In this tutorial. To show how Ajax works, we’ll create a JavaScript program that fetches data from an external API, format it, and return random Chuck Norris’ Jokes. It’s pretty simple, Ajax goes out, gets the joke, returns them, and then displays them to the user.
+
+### Prerequisites
+
+To followup this tutorial, you will need to have a basic idea of the following :
+
+- [JavaScript.](https://www.geeksforgeeks.org/introduction-to-javascript/#:~:text=JavaScript%20is%20a%20lightweight%2C%20cross,well%20as%20Server%2Dside%20developments.)
```

Comment:
Please remove this part of the url: `#:~:text=JavaScript%20is%20a%20lightweight%2C%20cross,well%20as%20Server%2Dside%20developments`